docs/controls/Timeline.md

```diff
@@ -89,15 +89,17 @@ timelineDefault.loadContent(/* Data array D */);
-| locale        | object   | LOCALE.en_US                                  | Locale object for X-Axis tick values                                                 |
-| throttle      | number   | (1000/60) => time in ms                       | Delay between resize of a browser window                                             |
-| showLabel     | boolean  | true                                          | Toggle to show X axis label                                                          |
-| showLegend    | boolean  | true                                          | Toggle to show graph legend                                                          |
-| padding       | object   | `{ top: 10, bottom: 5, left: 30, right: 50 }` | Refer [Padding](Padding.md)                                                          |
-| pan           | object   | {}                                            | Refer [Panning](./Panning.md)                                                        |
+| Property Name          | Expected | Default                                        | Description                                                                          |
+| ---------------------- | -------- | ---------------------------------------------- | ------------------------------------------------------------------------------------ |
+| bindLegendTo           | string   | null                                           | If DOM id provided, binds legend into that container (Example: `"#legendContainer"`) |
+| locale                 | object   | LOCALE.en_US                                   | Locale object for X-Axis tick values                                                 |
+| throttle               | number   | (1000/60) => time in ms                        | Delay between resize of a browser window                                             |
+| showLabel              | boolean  | true                                           | Toggle to show X axis label                                                          |
+| showLegend             | boolean  | true                                           | Toggle to show graph legend                                                          |
+| padding                | object   | `{ top: 10, bottom: 10, left: 30, right: 50 }` | Refer [Padding](Padding.md)                                                          |
+| pan                    | object   | {}                                             | Refer [Panning](./Panning.md)                                                        |
+| removeContainerPadding | boolean  | false                                          | Toggle to remove carbon container top and bottom padding                             |
+| legendPadding          | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`     | Sets the legend padding and removes the carbon default margin of 8px around legend   |
```

Comment:
Nit: alphabetize

Reply:
Alphabetized the props [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/751bb4e91a16e2653e7d094c18ae089e4393678f).